operators/src/adapters/raster_subquery/raster_subquery_adapter.rs

```diff
@@ -671,7 +671,7 @@ mod tests {
         let query_ctx = MockQueryContext::test_default();
         let tiling_strat = exe_ctx.tiling_specification;
 
-        let op = mrs1.initialize(&exe_ctx).await.unwrap();
+        let op = mrs1.initialize(Default::default(), &exe_ctx).await.unwrap();
```

Comment:
would it make sense to have a `initialize_root` method or something like this, where you don't have to pass the Path?

Reply:
https://github.com/geo-engine/geoengine/pull/780/commits/3c24e10e4c1d13236bfc5c7f8ea90c23ef5d4b1b
https://github.com/geo-engine/geoengine/pull/780/commits/8bfcc0fe6f76ed5e4b55227de2aab7045b13686f
https://github.com/geo-engine/geoengine/pull/780/commits/408c8cb2485626355c1b7f4e855d224c1aea168c

Reply:
sorry I wasn't precise enough. I meant adding an `initialize_root` function on the `Operator` such that one don't has to specify the WorkflowOperatorPath at all, because it is implicit that the root operator has the root path.

Reply:
we could do that but i think the root should be initialized by the creator of the workflow. It might add the workflow id to it or something.